src/dependency-builder/build-tree.js

```diff
@@ -370,6 +328,7 @@ function updateTreeWithLinkFilesAndImportSpecifiers(tree: Tree, pathMap: PathMap
  */
 export default async function getDependencyTree(baseDir: string, consumerPath: string, filePaths: string[], bindingPrefix: string):
   Promise<{ missing: Object, tree: Tree}> {
+  const packageJson = PackageJson.findPackage(consumerPath);
```

Comment:
@amitgilad3  you still check only the consumer package.json.
We need to check the component dir package.json first then start bubble up.